reflex/model.py

```diff
@@ -40,13 +40,13 @@ def get_engine(url: Optional[str] = None):
         console.warn(
             "Database is not initialized, run [bold]reflex db init[/bold] first."
         )
-    echo_db_query = False
-    if conf.env == constants.Env.DEV and constants.SQLALCHEMY_ECHO:
-        echo_db_query = True
+    # Print the SQL queries if the log level is INFO or lower.
+    echo_db_query = console.LOG_LEVEL <= console.LogLevel.INFO
```

Comment:
```suggestion
    echo_db_query = os.environ.get("SQLALCHEMY_ECHO", False)
```

i don't think we should change the semantics of the query echo, because this can leak sensitive information, it really should only be on if the user explicitly requests it.